### ISSUE TITLE
NTR: Borrelia burgdorferi PCR in CSF, blood or synovium positive (Lab: bacteriology, virology, parasitology, fungal)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8065]
New term request Borrelia burgdorferi PCR in CSF, blood or synovium positive (samant_101219003423): Cerebrospinal fluid polymerase chain reaction; DNA  UMLS: 2362659

**Comment by pnrobinson** [comment]
@MickeySegal
adding a term for CSF -- please add new issues if more are needed

Adding new term: 
- ID: HP:6000548
- label: Positive CSF Borrelia burgdorferi nucleic acid test
- parents: [Positive CSF nucleic acid pathogen test (HP:0430140)](https://hpo.jax.org/app/browse/term/HP:0430140)
- pmid: 
- def: Detection of Borrelia burgdorferi nucleic acid in the cerebrospinal fluid (CSF) by a method such as polymerase chain reaction.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8065
- synonyms:
